staging/src/k8s.io/apimachinery/pkg/util/wait/wait_test.go

```diff
@@ -455,9 +459,13 @@ func TestPollForever(t *testing.T) {
 				return
 			}
 		}
-		t.Fatalf("expected closed channel after two iterations")
+		t.Error("expected closed channel after two iterations")
 	}()
 	<-complete
+
+	if len(errc) != 0 {
+		t.Fatal(<-errc)
+	}
```

Comment:
with https://github.com/kubernetes/kubernetes/pull/99371/files#r634192402 we don't need thiis

Reply:
Think we need it as well. After `done` has been written in [L454](https://github.com/tiloso/kubernetes/blob/627218a8e2e8dddc70312c04b0fd5a90bccb8022/staging/src/k8s.io/apimachinery/pkg/util/wait/wait_test.go#L454) the `ConditionFunc` is being called a last time and `PollInfinite` might return an error.